components/navigation/JobPostItem.js

```diff
@@ -1,46 +1,87 @@
+import { join } from "lodash";
+
 export default function JobPostItem({ job }) {
-  return (    
+  return (
     <li>
       <a href={job.slug} className="block hover:bg-gray-50">
         <div className="px-4 py-4 sm:px-6">
           <div className="flex items-center justify-between">
+          <div className="flex items-center">
+          <img src={job.company.logoUrl} className="h-6 pr-0.5" /> 
```

Comment:
```suggestion
          <img src={job.company.logoUrl} className="h-6 mr-2" /> 
```

I'd increase the separation between the logo and the text. Also, to separate elements is usually better to use `margin` instead of `padding`, since we don't want to "pad" or "fill" the image object.

Reply:
ohk sure , do it as well